parseLoop is main script

aaiello30 opened the pull request:
parseLoop calls parseSetup, parseData, parsePlot to create and save
histogram graphs of each of the jobs in decorrelation.xlsx. parseLoop
also performs some mean, range, and error analysis. C4Model folder
contains subfolders for each of the jobs, each subfolder contains
histogram plots and subplot of all 10 frequencies. parseResults.xlsx
contains res data for each of the jobs/frequencies. res_vs_freq images
show res dependence on frequency. rng_avg_err.m contains variables such
as the range of res values, the average res value, and percent change.

aaiello30 committed:
parseLoop is main script

parseLoop calls parseSetup, parseData, parsePlot to create and save
histogram graphs of each of the jobs in decorrelation.xlsx. parseLoop
also performs some mean, range, and error analysis. C4Model folder
contains subfolders for each of the jobs, each subfolder contains
histogram plots and subplot of all 10 frequencies. parseResults.xlsx
contains res data for each of the jobs/frequencies. res_vs_freq images
show res dependence on frequency. rng_avg_err.m contains variables such
as the range of res values, the average res value, and percent change.

aaiello30 commented:
Hi Rogge (@waterfall414). Just testing this github system and commenting here. I've uploaded some matlab scripts/functions as well as a folder called C4Model which contains all the figures and results. Let me know if you get this

waterfall414 commented:
Thanks for the work! I can't merge it right now because there are some improvements I want on the code & plots.

waterfall414 commented:
Change the following lines:

% timemark = 0.5:11.6666:0.5+11.6666*10; % Can be improved here
tm_start = 1.5:11.666:1.5+11.666*9;
tm_end = 10.5:11.666:10.5+11.666*9;

% fr_mark = zeros(size(timemark)); 

fm_start = zeros(tm_start);
fm_end = zeros(tm_end);

for k = 1:11
    % fr_mark(k) = findClosest(timemark(k),xs+delay);
    fm_start = findClosest(fm_start,xs+delay);
    fm_end = findClosest(fm_end,xs+delay);
end

[~,~,res] = convert_data_struct(data);

% For the frequency group, find the start and end
% startingFrame = fr_mark(freqGroup);
startingFrame = fm_start(freqGroup);
% endingFrame = fr_mark(freqGroup+1);
endingFrame =fm_end(freqGroup);

aaiello30 committed:
update 5/4/2018

dataLoop updated with new cutoffs for determining frequency sections. Only histogram subplots are saved. Res data is reorganized by amplitude (1-4mm) and plots are made of res vs frequency and res vs average velocity.
